R/01-webdata-obj.R

```diff
@@ -161,3 +173,19 @@ setAs('list', 'webdata', function(from){
   return(.Object)
 })
 
+#' @param xml a GDP xml document
+#' @importFrom XML xmlRoot getNodeSet xmlValue xmlParse
+readGeojobXML <- function(xml) {
```

Comment:
This is very similar to inputs function I made for webprocess, you can probably just use that here. (file inputs-webprocess.R)